package.json

```diff
@@ -1,11 +1,28 @@
 {
   "name": "@divriots/dockit-react",
-  "version": "0.0.66",
+  "version": "0.0.67",
   "description": "",
   "license": "ISC",
   "type": "module",
   "homepage": "https://backlight.dev/preview/wzPn6NO54tDJlBxCtjGx/",
   "scripts": {
     "serve": "npx backlight@latest serve wzPn6NO54tDJlBxCtjGx --open"
+  },
+  "devDependencies": {
```

Comment:
See my comment in discord about devDependencies/dependencies distinction

Reply:
nitpick: put devDependencies below dependencies